india_compliance/gst_india/utils/gstr/gstr.py

```diff
@@ -0,0 +1,116 @@
+import frappe
+
+from india_compliance.gst_india.constants import STATE_NUMBERS
+from india_compliance.gst_india.doctype.inward_supply.inward_supply import (
+    create_inward_supply,
+)
+
+
+def get_mapped_value(value, map):
+    return map.get(value)
+
+
+class GSTR:
+    # Maps of API keys to doctype fields
+    KEY_MAPS = frappe._dict()
+
+    # Maps of API values to doctype values
+    VALUE_MAPS = frappe._dict(
```

Comment:
Shouldn't be a class var if it is designed to be mutated. Class vars are shared across all instances.